packages/block-editor/src/components/inserter/block-patterns-tab/pattern-category-previews.js

```diff
@@ -49,6 +50,10 @@ export function PatternCategoryPreviews( {
 	const [ patternSyncFilter, setPatternSyncFilter ] = useState( 'all' );
 	const [ patternSourceFilter, setPatternSourceFilter ] = useState( 'all' );
 
+	// Move to zoom out mode when this component is mounted
+	// and back to the previous mode when unmounted.
+	useZoomOut();
```

Comment:
I'd much rather this not be a `useEffect`. If you refactor it to call the action in the same `onClick` for switching to the pattern category previews would be better.

Reply:
yes I too dislike this hook 😁